vivarium/core/engine.py

```diff
@@ -423,8 +424,18 @@ def __init__(
                 provide as the value for the key ``experiment_id``.
             display_info: prints experiment info
             progress_bar: shows a progress bar
-            emit_config: If True, this will emit the serialized
-                processes, topology, and initial state.
+            store_emit: An optional dictionary to turn emits on or off. This
+                dictionary requires the keys (`on`,`off`), mapping to a list
```

Comment:
```suggestion
            store_emit: An optional dictionary to turn emits on or off. This
                dictionary may contain the keys `on` and `off`, mapping to a list
```

Each key is optional, right? You could provide an empty dict, in which case nothing would happen